sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -45,31 +48,27 @@ class TableAccessPolicy(GenAccessPolicy):
     fields are missing, the request will fail. Likewise, if a field is specified
     both in the Shared Access Signature URL and in the stored access policy, the
     request will fail with status code 400 (Bad Request).
+    """
 
-    :keyword str permission:
```

Comment:
These should still be documented as keyword params - because this object is both input and output.

Reply:
Same comment applies to all models that customers might need to construct.

Reply:
Added keyword params into docstring in constructor for all public models. And removed class properties docstring to avoid duplicate description, not sure if that's proper.